dockstore-cli-integration-testing/pom.xml

```diff
@@ -46,6 +46,18 @@
             <groupId>io.dockstore</groupId>
             <artifactId>dockstore-client</artifactId>
             <version>1.12.0-alpha.0-SNAPSHOT</version>
+            <exclusions>
```

Comment:
What is the version used in the bom?

Reply:
<jackson.version>2.12.3</jackson.version>
https://github.com/dockstore/dockstore/blob/b08519d93ae891f59ccca53b666bf629c9e2d0d3/bom-internal/pom.xml#L73